### PR TITLE
Hw 03

### PR DESCRIPTION
**Pull request by KuntsevichKiryl**

### CODE REVIEW

File: hw_03/HW03_01.Operators/Program.cs [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+                return double.NaN;
+            }
+        }
+
+        private static double Sum(double a, double b)
+        {
+            return a + b;
+        }
+    }
+}
```
Comment (VadzimPapko) [inline hw_03/HW03_01.Operators/Program.cs:64]: correct

File: hw_03/HW03_05.Alphabet/Program.cs [hunk 2]
```diff
@@ -0,0 +1,15 @@
+﻿using System;
+
+namespace HW03_05.Alphabet
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            for (byte n = 90; n >= 65; n--)
```
Comment (VadzimPapko) [inline hw_03/HW03_05.Alphabet/Program.cs:9]: 90, 65 - magic numbers

File: hw_03/HW03_05.Alphabet/Program.cs [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+            {
+                Console.WriteLine((char)n);
+            }
+        }
+    }
+}
```
Comment (VadzimPapko) [inline hw_03/HW03_05.Alphabet/Program.cs:15]: in general correct

File: hw_03/HW03_06.Movement/Program.cs [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+                case "EXIT":
+                    System.Environment.Exit(0);
+                    break;
+                default:
+                    Console.WriteLine("stay");
+                    break;
+            }
+        }
+    }
+}
```
Comment (VadzimPapko) [inline hw_03/HW03_06.Movement/Program.cs:48]: well done

File: hw_03/HW03_07.Birthday/Program.cs [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+            if (year > yearnow || month > 12 || month == 0 || monthnow > 12 || monthnow == 0)
+            {
+                Console.WriteLine("incorrect date");
+                return;
+            }
+            uint age = month - monthnow < 0 ? yearnow - year : yearnow - year - 1;
+            Console.WriteLine($"Age: {age}");
+        }
+    }
+}
```
Comment (VadzimPapko) [inline hw_03/HW03_07.Birthday/Program.cs:26]: correct!

File: hw_03/HW03_09.Calculator/Program.cs [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+            }
+
+            else
+            {
+                Console.WriteLine("Incorrect radius");
+                return double.NaN;
+            }
+
+        }
+
```
Comment (VadzimPapko) [inline hw_03/HW03_09.Calculator/Program.cs]: remove this line

File: hw_03/HW03_09.Calculator/Program.cs [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+    }
+
```
Comment (VadzimPapko) [inline hw_03/HW03_09.Calculator/Program.cs]: remove this line

File: hw_03/HW03_09.Calculator/Program.cs [hunk 8]
```diff
@@ -0,0 +1,65 @@
...
+            {
+                return Math.PI * Math.Pow(r, 2);
+            }
+
+            else
+            {
+                Console.WriteLine("Incorrect radius");
+                return double.NaN;
+            }
+
```
Comment (VadzimPapko) [inline hw_03/HW03_09.Calculator/Program.cs]: remove this line

File: hw_03/HW03_09.Calculator/Program.cs [hunk 9]
```diff
@@ -0,0 +1,65 @@
...
+        {
+            return a % b;
+        }
+        public static double Square(string n)
+        {
+            if (double.TryParse(n, out double r) && r >= 0)
+            {
+                return Math.PI * Math.Pow(r, 2);
+            }
+
```
Comment (VadzimPapko) [inline hw_03/HW03_09.Calculator/Program.cs]: remove this line

File: hw_03/HW03_09.Calculator/Program.cs [hunk 10]
```diff
@@ -0,0 +1,65 @@
+﻿using System;
+
+namespace HW03_09.Calculator
+{
+    class Program
```
Comment (VadzimPapko) [inline hw_03/HW03_09.Calculator/Program.cs:5]: correct

**Review by VadzimPapko — COMMENTED** [review]
Reviewed. Your score is 9

**Comment by VadzimPapko** [comment]
@KuntsevichKiryl we need to fix merge conflict

**Comment by KuntsevichKiryl** [comment]
> @KuntsevichKiryl we need to fix merge conflict

merge done

**Pull request merged**